### ISSUE TITLE
Your Kanboard instance is not configured to install plugins from the user interface

### ISSUE THREAD

**Issue by timdonovanuk** [kanboard/kanboard#4894]
- All the previously closed bug reports on this links to documentation that is now 404
- I have spent 2 mins searching the documentation but cannot see a section on this
- Don't tell users "it's not configured" - empower them to fix it by telling them HOW to configure it
- Why not have this enabled/setup by default? I used docker to install

**Comment by fguillot** [comment]
See https://docs.kanboard.org/en/latest/admin_guide/plugins.html

![image](https://user-images.githubusercontent.com/323546/138534438-c4984c50-37be-4737-840e-24d16367d66d.png)

**Comment by deeagle** [comment]
@voodoohoo Do you link the correct `config.php` in `docker-compose.yml`?

Because I maped the false one with `config.default.php` (and I do all changes in `config.php`).

After that I have to adapt the permissions to **777**.
Because the permissions of the folder was **755 _apt system-journal**.

**Comment by laundmo** [comment]
I have to agree with OP here, getting this working is quite unintuitive. I would love for this to be easier to set up with docker, by just setting a env var for example. Then, in the docs, make that envvar set to false by default. but set it explicitly. that way just looking at the docker command or compose its obvious what needs to be done.

**Comment by sbn-purchark** [comment]
https://kanboard.discourse.group/t/docker-and-plugins/707/2

He is correct, the changed suggested works.

**Comment by HardlineStudios** [comment]
> https://kanboard.discourse.group/t/docker-and-plugins/707/2
> 
> He is correct, the changed suggested works.

Yeah this worked for me as well.

**Comment by pannet1** [comment]
i am not using docker and i was changing the config.default.php whereas, it should be copying as config.php and the changes should be made there. loved kanboard so far, has the potential to be hacked as the top project management tool with the plugins.
